jenkins/release-tag/release-tag.jenkinsfile

```diff
@@ -1,6 +1,9 @@
 lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
 
 pipeline {
+    options {
+            timeout(time: 2, unit: 'HOURS')
```

Comment:
How long does this job take, in your tests?

Reply:
It depends on multiple things: the number of components the manifest, the size of contents for each repo, github fetching time, etc.. I tested with only three components and it normally took around 2 mins.

Reply:
Then let's change the timeout to be 30mins?

Reply:
I think the timeout here is the max time this job possibly takes. I would like to have this longer just in case. Shouldn't be a big deal IMO.

Reply:
2 hr is a ok time range, I normally have 2x 3x time for max range.

Reply:
3x for 10min is 30mins

Reply:
We won't know if this takes 10 mins or 30 mins or even more. I believe 2 hours should be an acceptable range.

Reply:
I feel under no circumstance it would take 2 hrs, or even 1hr. It just blocks the node unnecessarily and therefore can be reduced. It's not a blocker for me, but timeouts should be visited carefully as they are the failsafes that unblock nodes in case a job is stuck